From: hololeap <hololeap@protonmail.com>
Signed-off-by: hololeap <hololeap@protonmail.com>

Adding:
* Run `haskell-language-server` and `haskell-language-server-wrapper`
  with `+RTS -N` ghc flags allowing for parallelism on systems
  that can support it.

--- haskell-language-server-1.4.0.0/haskell-language-server.cabal
+++ haskell-language-server-1.4.0.0-new/haskell-language-server.cabal
@@ -323,7 +323,7 @@
     -rtsopts
     -- disable idle GC
     -- increase nursery size
-    "-with-rtsopts=-I0 -A128M"
+    "-with-rtsopts=-I0 -A128M -N"
     -Wno-unticked-promoted-constructors
   if flag(pedantic)
     ghc-options: -Werror
@@ -391,7 +391,7 @@
     -rtsopts
     -- disable idle GC
     -- increase nursery size
-    "-with-rtsopts=-I0 -A128M"
+    "-with-rtsopts=-I0 -A128M -N"
   if flag(pedantic)
     ghc-options: -Werror
 
